base_layer/wallet_ffi/src/lib.rs

```diff
@@ -2628,6 +2626,7 @@ pub unsafe extern "C" fn comms_config_create(
             match ni {
                 Ok(ni) => {
                     let config = TariCommsConfig {
+                        network: Network::Weatherwax,
```

Comment:
Should this not rather be an option, rather than a hardcoded value

Reply:
I agree, however this is how it was previously so changing it is out of scope of this PR.